astro/src/content/docs/get-started/core-concepts/users.mdx

```diff
@@ -99,14 +99,28 @@ These are the default fields that are available when creating a user. You can ad
       Then only the setup password message will be sent to the user. Setting up the password using the message sent during this user create operation will verify the user’s email address or phone number if it is not already verified.
     </Aside>
     <Aside>
-      This is only available if the [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) in the tenant Template settings is set up.
+      <InlineField>Send password setup to</InlineField> is only available if the following tenant settings are:
+      * [Setup password option](/docs/get-started/core-concepts/tenants#template-settings) - template settings must be set up
+      * [Password enabled](/docs/get-started/core-concepts/tenants#password) - set to `true`
     </Aside>
   </APIField>
   <APIField name="Password" required>
-    The password of the user. This field is only displayed if <InlineField>Send email to set up password</InlineField> is not enabled.
+    The password of the user.
+
+    <Aside>
+      This field is only displayed if:
+      * <InlineField>Send password setup to</InlineField> is not enabled or is set to `Do not send`
```

Comment:
Earlier references to `<InlineField>Send email to set up password</InlineField>` were missed. Fixed that.